src/backends/perf_pt/mod.rs

```diff
@@ -616,6 +612,22 @@ mod tests {
         (ptxed_args, vdso_tempfile)
     }
 
+    /*
+     * Determine if the given x86_64 assembler mnemonic should terminate a block.
+     *
+     * Mnemonic assumed to be lower case.
+     */
+    fn instr_terminates_block(instr: &str) -> bool {
+        assert!(!instr.starts_with("["));
```

Comment:
Why is this assert here?

Reply:
It's debug code. I'll kill it.

(this branch predates the panic change).